### PR TITLE
Add preinstallLtw parameter

### PR DESCRIPTION
**Pull request by cikelengfeng**
In build-msal-automation-app.yml template (android-complete\msal\azure-pipelines\ui-automation\templates), we should add -PpreInstallLtw=$(parameters. preInstallLtw) in “Assemble MSAL Automation App” and “Assemble MSAL Automation App Instrumented Tests”. Also add the variable in the parameters.  Note : This is not a separate pipeline, it is a template used in the [UI automation pipeline](https://identitydivision.visualstudio.com/Engineering/_build?definitionId=1490&_a=summary) 

Same as above step in build-broker-automation-app.yml template file as well. 

Next, in broker-test.yml file ([UI automation pipeline](https://identitydivision.visualstudio.com/Engineering/_build?definitionId=1490&_a=summary)), wherever we use the above two templates, pass the parameter preInstallLtw: ${{ parameters. preInstallLtw }} 

This means that we can set this flag when we trigger the pipeline. Verify if the pipeline succeeds when the flag is true and false. All the existing testcases should succeed in both cases. If some testcase fails, please feel free to ask help from Broker team on why this could affect an existing testcase.

### CODE REVIEW

File: azure-pipelines/ui-automation/templates/build-broker-automation-app.yml [hunk 1]
```diff
@@ -64,6 +67,9 @@ jobs:
           if (("${{ parameters.commonVersion }}" -ne "")) {
               $assembleTask = $assembleTask + " -PdistCommonVersion=" + "${{ parameters.commonVersion }}"
           }
+          if ("${{ parameters.preinstallLtw }}" -eq "True") {
+              $assembleTask = $assembleTask + " -PpreinstallLtw=true"
```
Comment (fadidurah) [inline azure-pipelines/ui-automation/templates/build-broker-automation-app.yml:71]: preInstallLtw*

File: azure-pipelines/ui-automation/templates/build-broker-automation-app.yml [hunk 2]
```diff
@@ -82,6 +88,9 @@ jobs:
           if (("${{ parameters.commonVersion }}" -ne "")) {
               $assembleTestTask = $assembleTestTask + " -PdistCommonVersion=" + "${{ parameters.commonVersion }}"
           }
+          if ("${{ parameters.preinstallLtw }}" -eq "True") {
+              $assembleTestTask = $assembleTestTask + " -PpreinstallLtw=true"
```
Comment (fadidurah) [inline azure-pipelines/ui-automation/templates/build-broker-automation-app.yml:92]: preInstallLtw*

File: azure-pipelines/ui-automation/templates/build-broker-automation-app.yml (hunk 1 above)
Comment (fadidurah) [reply azure-pipelines/ui-automation/templates/build-broker-automation-app.yml:71]: you have a lower case i in "install", build won't recognize this as it's case sensitive.

Comment (fadidurah) [reply azure-pipelines/ui-automation/templates/build-broker-automation-app.yml:71]: Please also consider this for all other instances of this parameter

Comment (cikelengfeng) [reply azure-pipelines/ui-automation/templates/build-broker-automation-app.yml]: Fixed.

**Review by fadidurah — APPROVED** [review]
LGTM

**Pull request merged**